templates/endpoints/README.md

```diff
@@ -151,6 +153,56 @@ Endpoints allows you to easily add new models by adding a single configuration f
 To learn more about how to customize or add new models, 
 see the [Model Registry](models/README.md).
 
+# Serving LoRA Models
+
+`serve_lora.yaml` and `query_lora.py` are provided for you in this template. We support serving multiple LoRA adapters with a common base model in the same request batch. In addition, we use Serve multiplexing to reduce the number of swaps for LoRA adapters.
+
+Make sure you replace `dynamic_lora_loading_path` and `HUGGING_FACE_HUB_TOKEN` config in `serve_lora.yaml` with your own values. And place the LoRA checkpoint in the `dynamic_lora_loading_path` bucket.
+
+To deploy the LoRA models, run:
+```shell
+serve run serve_lora.yaml
+```
+
+This will take up to a minute or so to load depending on the model size given the required worker type is already up.
+For dynamic LoRA models, make sure to have your LoRA checkpoint stored in `{base_path}/{base_model_id}:{suffix}:{id}` and change the model id accordingly in `query_lora.py`.
+
+To query the LoRA model, run:
+```shell
+python query_lora.py
+
+# Example output:
+# {
+#     "id": "meta-llama/Llama-2-7b-chat-hf:lora-model:1234-472e56b56039273c260e783a80950816",
+#     "object": "text_completion",
+#     "created": 1699563681,
+#     "model": "meta-llama/Llama-2-7b-chat-hf:lora-model:1234",
+#     "choices": [
+#         {
+#             "message": {
+#                 "role": "assistant",
+#                 "content": " Sure, I can do that! Based on the target sentence you provided, I will construct the underlying meaning representation of the input sentence as a single function with attributes and attribute values.\n\nThe function I have constructed is:\n\n['inform', 'available_on_steam'] [1] [developer] [Slightly Mad Studios] [/]  \n\nThe attributes are:\n\n[1] [release_year] [2012]\n[developer] [Slightly Mad Studios]"
+#             },
+#             "index": 0,
+#             "finish_reason": "stop"
+#         }
+#     ],
+#     "usage": {
+#         "prompt_tokens": 285,
+#         "completion_tokens": 110,
+#         "total_tokens": 395
+#     }
+# }
+```
+
+A few tips for serving LoRA models:
+1. LoRA base models should be passed in the serve config file `serve_lora.yaml` in the `multiplex_models` config.
```

Comment:
Numbering still seems off

Reply:
This will render correctly in markdown 

something like 
```
1. foo
1. bar
1. baz
```
will turn into 
1. foo
1. bar
1. baz 
😄